code/ZAS/XGM_reactions.dm

```diff
@@ -27,19 +27,20 @@
 	name = "Cryotheum-Oxygen Reaction"
 
 /datum/gas_reaction/cryotheum_oxygen_reaction/reaction_is_possible(datum/gas_mixture/mixture)
-	return mixture[GAS_CRYOTHEUM] > 0 && mixture[GAS_OXYGEN] > 0
+	return mixture[GAS_CRYOTHEUM] > 0 && mixture.molar_density(GAS_CRYOTHEUM) * CELL_VOLUME > MOLES_CRYOTHEUM_VISIBLE && mixture[GAS_OXYGEN] > 0
```

Comment:
For consistency with other ZAS code you may want to use 'QUANTIZE(molar_density(GAS_CRYOTHEUM))' (rounds to 0.0001) to avoid rounding issues.